src/components/common/Details/DetailApp.js

```diff
@@ -64,15 +63,19 @@ const DetailApp = (props: Props) => {
         <li className="item">
           <span className="item-name">Daily Spend Account</span>
           <span className="item-value">
-              <a href={`/${SMART_WALLET}/126812`} onClick={(e) => viewStore.linkHandler(e, SMART_WALLET, '126812')}>
-                0x012345567abcdef012345567abcdef012345567abcdef012345567abc
-                <CopyButton value="0x012345567abcdef012345567abcdef012345567abcdef012345567abc" />
-              </a>
-            </span>
+            <a href={`/${SMART_WALLET}/126812`} onClick={(e) => viewStore.linkHandler(e, SMART_WALLET, '126812')}>
+              0x012345567abcdef012345567abcdef012345567abcdef012345567abc
+              <CopyButton value="0x012345567abcdef012345567abcdef012345567abcdef012345567abc" />
+            </a>
+          </span>
         </li>
         <li className="item">
           <span className="item-name">Master Accounts</span>
-          <span className="item-value">2 SMH</span>
+          <span className="item-value">
+            <a>
```

Comment:
wrong tag a - see above, also add e.preventDefault() onClick method for stoping propagation, all links in this page 
shouldn't work, it's just mock data

Reply:
done